src/main/java/htsjdk/variant/variantcontext/VariantContextBuilder.java

```diff
@@ -342,23 +356,52 @@ private void makeAttributesModifiable() {
      * @return this builder
      */
     public VariantContextBuilder filters(final Set<String> filters) {
-        this.filters = filters;
+        makeFiltersModifiable();
+        if (filters == null) {
+            unfiltered();
```

Comment:
This nulls out `filters`; if this path is taken than there's no point in calling `makeFiltersModifiable()`

Reply:
👍